src/elements/content-preview/stories/ContentPreview.stories.js

```diff
@@ -87,6 +87,26 @@ export const PreviewWithSidebar = () => {
     );
 };
 
+export const PreviewWithBoxAI = () => {
+    const fileId = text('File ID', global.FILE_ID);
+    const token = text('Access Token', global.TOKEN);
+
+    return (
+        <IntlProvider locale="en">
+            <ContentPreview
+                contentAnswersProps={{
+                    show: true,
```

Comment:
do we want to enable this yet in storybook since we're still implementing?

Reply:
This is the easiest way to test the feature itself, I dont see much of an issue since people can see the PRs go through and we dont advertise storybook on the docs